packages/rest/src/schemas/CredentialProposalRequest.ts

```diff
@@ -0,0 +1,29 @@
+import { CredentialPreview } from "@aries-framework/core";
+import { Type } from "class-transformer";
+import { IsString, ValidateNested, IsOptional, Matches } from "class-validator";
+
+export class CredentialProposalRequest {
+  @IsOptional() @IsString() @Matches(/^(did:sov:)?[a-zA-Z0-9]{21,22}$/) public schema_issuer_did?: string;
```

Comment:
I prefer to format decorators like this:

```suggestion
  @IsOptional()
  @IsString()
  @Matches(/^(did:sov:)?[a-zA-Z0-9]{21,22}$/)
  public schema_issuer_did?: string;
```